modules/ocf/manifests/packages/grafana/apt.pp

```diff
@@ -0,0 +1,13 @@
+class ocf::packages::grafana::apt {
+  apt::key { 'grafana':
+    id     => '418A7F2FB0E1E6E7EABF6FE8C2E73424D59097AB',
+    source => 'https://packagecloud.io/gpg.key',
+  }
+
+  apt::source { 'grafana':
+    location => 'https://packagecloud.io/grafana/stable/debian/',
```

Comment:
Using just http should be fine here too, since the packages are signed, but we should also already have `apt-transport-https` installed so this still works. Might want to add that package as a dependency here if you use https?